server/public/model/feature_flags.go

```diff
@@ -85,6 +87,7 @@ func (f *FeatureFlags) SetDefaults() {
 	f.WysiwygEditor = false
 	f.OnboardingTourTips = true
 	f.CloudReverseTrial = false
+	f.DataRetentionConcurrencyEnabled = false
```

Comment:
I would expect this to default `true` to avoid altering the existing behaviour for anyone upgrading. Alternatively, invert the logic and rename it to `DataRetentionConcurrencyDisabled`.

Reply:
Done!